### DISCUSSION TITLE
Windows 11 Install does nothing

### DISCUSSION THREAD

**Discussion by steven86m** [rustdesk/rustdesk#6893] [General]
### Bug Description

Trying to install on windows 11 and it does nothing even as install as admin

### How to Reproduce

Install the file as normal

### Expected Behavior

Does nothing, no install

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

N/A

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/6786
